packages/react-dev-utils/FileSizeReporter.js

```diff
@@ -30,7 +30,7 @@ function printFileSizesAfterBuild(webpackStats, previousSizeMap) {
       var previousSize = sizes[removeFileNameHash(root, asset.name)];
       var difference = getDifferenceLabel(size, previousSize);
       return {
-        folder: path.join('build', path.dirname(asset.name)),
+        folder: path.join(buildFolder, path.dirname(asset.name)),
```

Comment:
This prints the whole path which is unfortunate.
Can we take just the folder name from it?
`path.basename` might do it.